spec/30-processing-model.md

```diff
@@ -0,0 +1,52 @@
+# Processing Model
+
+This section provides a step-by-step description of the behavior of a tracing implementation - also called tracer -  when a request is received. This description can be used as a reference when implementing a Trace Context compliant tracing system, middleware or cloud service.
+
+## Processing Model for Working with Trace Context
+
+The processing model describes the behaviour of a tracer which forwards **and** modifies
+Trace Context headers.
+
+1. The tracer receives a `traceparent` header and may receive a `tracestate` header.
+2. The tracer tries to parse the version of the `traceparent` header.
+   - If the version prefix cannot be parsed, the tracer creates a new `traceparent`
+     header and removes all entries from `tracestate`.
+   - If the version number is higher than supported by the tracer,
+  the implementation uses the format defined in this specification to parse
+  `trace-id` and `parent-id`. The tracer will only parse `flags` values
+  supported by the current version of this specification and ignore all other
+  values. If parsing fails, the tracing system creates a new `traceparent` header
+  and removes all entries from `tracestate`.
+
+3. If the tracer supports the version number it validates `trace-id`
+and `parent-id`.
+   - If either `trace-id`, `parent-id` or `flags`  are invalid,  the tracer
+   creates a new `traceparent` header and removes all entries from `tracestate`.
+
+4. The tracer validates the `tracestate` header. If the `tracestate` header cannot be parsed the system creates a new empty `tracestate` header.
+
+5. For each outgoing request the tracer performs the following steps:
+
+   - The tracing system MUST modify the `traceparent` header.
```

Comment:
I believe the list of items beneath this item (Update `parent-id`, Update `recorded`, ..., etc) would benefit from being an ordered list, if we can get the markdown to comply.

Reply:
Order lists will switch to using lower case i on the second level. Maybe @plehegar know how to make nested ordered lists work.